app/Traits/Log/Severity.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Traits;
+
+trait Severity {
+    // 0       Emergency: system is unusable
```

Comment:
Maybe add the RFC this is based on [RFC 5424 section 6.2.1](https://www.rfc-editor.org/rfc/rfc5424#section-6.2.1)